blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubPipelineUpdateRequest.java

```diff
@@ -55,6 +55,7 @@ public BluePipeline update(BluePipeline pipeline) throws IOException {
                 folder.scheduleBuild(new Cause.UserIdCause());
             }
         }
+        item.save();
```

Comment:
and again: <img src='https://s-media-cache-ak0.pinimg.com/originals/d4/09/cd/d409cd68d10bf1726148d86d9bbaa95b.jpg'>